.azurePipeline/getPodsLogs.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+"""
+The script will currently be used in the Azure Pipeline to get back the logs of the pods when a test is failing
+allowing developers to document the issue.
+"""
+
+import json
+from pathlib import Path
+import subprocess
+import sys
+
+# The namespace is not configurable as we are expected to run this script only in Azure Dev-Ops which deployed the
+# service only in this namespace.
+NAMESPACE = "test-azure"
+
+
+def get_list_pods(_release_name):
+    cmd = "kubectl --namespace {} get pods -l release={} -o json".format(NAMESPACE, _release_name)
+    r = subprocess.run(cmd.split(" "), capture_output=True)
+    items_from_release = json.loads(r.stdout)['items']
+    return items_from_release
+
+
+def get_logs_container(container_info, file, previous=False):
+    cmd = "kubectl --namespace {} logs {} {}".format(NAMESPACE, container_info['full_pod_name'], container_info['name'])
+    if previous:
+        cmd += " --previous"
+    subprocess.call(cmd.split(" "), stdout=file)
+
+
+def info_from_pod(_pod, _directory):
+    pod_name = _pod.get('metadata').get('name')
+    list_containers = _pod.get('status').get('containerStatuses')
+    for container in list_containers:
+        name_container = container.get('name')
+        restart_count = container.get('restartCount')
+        image_name = container.get('image')
+        info = {'full_pod_name': pod_name, 'short_pod_name': pod_name[len('benchmark-es-data61-xyz-'):],
+                'name': name_container, 'image': image_name, 'restart_count': restart_count}
+        
+        with open(_directory / 'pod-{}_container-{}.log'.format(info['short_pod_name'], info['name']), 'wt') as f:
+            json.dump(info, f, indent=2, sort_keys=True)
+            f.write('\n\n')
+        with open(_directory / 'pod-{}_container-{}.log'.format(info['short_pod_name'], info['name']), 'at') as f:
+            get_logs_container(info, f)
```

Comment:
Mixing json and logging output is a bit strange

Reply:
But the information is relevant.
From 
```
{
  "full_pod_name": "esdf764b0f-memstore-server-0",
  "image": "redis:4.0.11-alpine",
  "name": "redis",
  "restart_count": 0,
  "short_pod_name": "er-0"
}

[logs]
```
would
```
"full_pod_name": "esdf764b0f-memstore-server-0",
"image": "redis:4.0.11-alpine",
"name": "redis",
"restart_count": 0,
"short_pod_name": "er-0"
----------------------------------------------------------------

[logs]
```
be better?
The most interesting from it it the image and the restart count. The other information are more relevant if the deployment was staying up.

Reply:
Sure the info is great. It was more of an observation, no need to change it.